documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -2607,12 +2580,19 @@ database.history.consumer.ssl.truststore.password=test1234
 database.history.consumer.ssl.key.password=test1234
 ----
 
-See link:{link-kafka-docs}.html[the Kafka documentation] for more details about _pass-through_ properties.
+As is the case with the xref:{context}-pass-through-configuration-properties-for-database-drivers[pass-through properties for database drivers], {prodname} strips the prefixes from the properties before {prodname} it passes them to configure the database history producer or consumer client.
+
+See the Kafka documentation for more details about link:https://kafka.apache.org/documentation.html#producerconfigs[Kafka producer configuration properties] and link:https://kafka.apache.org/documentation.html#consumerconfigs[Kafka consumer configuration properties].
+
+[id="{context}-database-history-properties-for-configuring-connector-behavior"]
+==== Database history properties for configuring connector behavior
+
+In addition to the pass-through database history properties that you use to configure producers or consumers, there is a set of `database.history.*` properties whose names do not include `producer` or `consumer`.
```

Comment:
Sorry, this was confusing, because, per my earlier comment, I hadn't yet removed this section, but just commented it out, so it still shows in the diff.